### DISCUSSION TITLE
Kein Dashboard ausserhalb vom Heimnetz

### DISCUSSION THREAD

**Discussion by Amensgermany** [rdmtc/RedMatic#581] [Q&A]
Hallo, seit dem Raspberrymatic-Update 3.71.12.20231015/3.71.12.20231020 komme ich nur noch im Heimnetz auf mein Dashboard. Hat sich da in der Architektur was grundlegendes geändert? Normalerweise läuft das von ausserhalb ja auch über Cloudmatic. Falls es an Raspberrymatic liegen sollte, könnte ich es im Raspi-Forum auch mal einstellen.
[redmatic.2023-10-20T19 08 35.008Z.log](https://github.com/rdmtc/RedMatic/files/13058053/redmatic.2023-10-20T19.08.35.008Z.log)

**Comment by hobbyquaker** [comment]
Die CCU (und damit Node-RED/Dashboard) sollte nie direkt aus dem Internet erreichbar sein - siehe https://github.com/rdmtc/RedMatic/wiki/Sicherheit; Fernzugriff nur per VPN. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
